apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/ChatModel.kt

```diff
@@ -62,6 +62,7 @@ object ChatModel {
   // current chat
   val chatId = mutableStateOf<String?>(null)
   val chatItems = mutableStateOf(SnapshotStateList<ChatItem>())
+  val chatViewMode = mutableStateOf(ChatViewMode.Message)
```

Comment:
why does it have to be a global variable, why couldn't it be a state variable local to ChatView passed to ChatInfoView?